src/service.js

```diff
@@ -245,6 +246,9 @@ class Service {
 		action.cache = action.cache !== undefined ? action.cache : (this.settings.$cache || false);
 		action.handler = this.Promise.method(handler.bind(this));
 
+		// action metrics options by default
+		action.metrics = defaultsDeep({ params: false, meta: true });
```

Comment:
`defaultsDeep` is necessary? I think it can work without it as well.

Reply:
Sorry, this is a mistake, it would be as follow:

defaultsDeep(action.metrics, {params: false,meta: true});

In order to guarantee the defaults value are these values for all actions.